lib/experimental/fonts/font-library/class-wp-rest-font-faces-controller.php

```diff
@@ -391,19 +408,46 @@ public function delete_item( $request ) {
 	 * @param WP_REST_Request $request Request object.
 	 * @return WP_REST_Response Response object.
 	 */
-	public function prepare_item_for_response( $item, $request ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable -- required by parent class
-		$data = array();
+	public function prepare_item_for_response( $item, $request ) {
+		$fields = $this->get_fields_for_response( $request );
+		$data   = array();
+
+		if ( rest_is_field_included( 'id', $fields ) ) {
+			$data['id'] = $item->ID;
+		}
+		if ( rest_is_field_included( 'theme_json_version', $fields ) ) {
+			$data['theme_json_version'] = 2;
+		}
 
-		$data['id']                 = $item->ID;
-		$data['theme_json_version'] = 2;
-		$data['parent']             = $item->post_parent;
-		$data['font_face_settings'] = $this->get_settings_from_post( $item );
+		if ( rest_is_field_included( 'parent', $fields ) ) {
+			$data['parent'] = $item->post_parent;
+		}
+
+		if ( rest_is_field_included( 'font_face_settings', $fields ) ) {
+			$data['font_face_settings'] = $this->get_settings_from_post( $item );
+		}
+
+		$context = ! empty( $request['context'] ) ? $request['context'] : 'edit';
+		$data    = $this->add_additional_fields_to_object( $data, $request );
+		$data    = $this->filter_response_by_context( $data, $context );
 
 		$response = rest_ensure_response( $data );
-		$links    = $this->prepare_links( $item );
-		$response->add_links( $links );
 
-		return $response;
+		if ( rest_is_field_included( '_links', $fields ) || rest_is_field_included( '_embedded', $fields ) ) {
+			$links = $this->prepare_links( $item );
+			$response->add_links( $links );
+		}
+
+		/**
+		 * Filters the font face data for a REST API response.
+		 *
+		 * @since 6.5.0
+		 *
+		 * @param WP_REST_Response $response The response object.
+		 * @param WP_Post          $post     Font face post object.
+		 * @param WP_REST_Request  $request  Request object.
+		 */
+		return apply_filters( 'rest_prepare_wp_font_face', $response, $item, $request );
```

Comment:
Adding each response property conditionally, so it can be toggled with the `_fields` param.